virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -0,0 +1,266 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="virt-4-15-release-notes"]
+= {VirtProductName} release notes
+include::_attributes/common-attributes.adoc[]
+:context: virt-4-15-release-notes
+
+toc::[]
+
+[id="virt-inclusive-language"]
+== Making open source more inclusive
+
+Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[our CTO Chris Wright's message].
+
+[id="virt-doc-feedback"]
+== Providing documentation feedback
+
+To report an error or to improve our documentation, log in to your link:https://issues.redhat.com[Red Hat Jira account] and submit a link:https://issues.redhat.com/secure/CreateIssueDetails!init.jspa?pid=12323181&issuetype=1&components=12333768&priority=10200&summary=%5BDoc%5D&customfield_12316142[Jira issue].
+
+[id="virt-about-cnv"]
+== About Red Hat {VirtProductName}
+
+Red Hat {VirtProductName} enables you to bring traditional virtual machines (VMs) into {product-title} where they run alongside containers, and are managed as native Kubernetes objects.
+
+{VirtProductName} is represented by the image:virt-icon.png[{VirtProductName},40,40] icon.
+
+You can use {VirtProductName} with either the xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes] or the xref:../../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[OpenShiftSDN] default Container Network Interface (CNI) network provider.
+
+Learn more about xref:../../virt/about_virt/about-virt.adoc#about-virt[what you can do with {VirtProductName}].
```

Comment:
Looks like the enterprise-4.15 version of about-virt has the placeholder text in it; we should fix that.

Reply:
Good catch! I am almost tempted to leave the placeholder text in because in a few weeks, I'd have to add it back in as part of the asynchronous release process! :-)